api/src/kots_app/kots_app_store.ts

```diff
@@ -612,6 +612,24 @@ export class KotsAppStore {
     }
   }
 
+  async getPreviouslyDeployedSequence(appId: string, clusterId: string, currentSequence: number): Promise<number | undefined> {
+    const q = `select max(sequence) as prev from app_downstream_version where app_id = $1 and cluster_id = $2 and sequence < $3`;
```

Comment:
should this order by applied_at, and not sequence?

to account for rollbacks

Reply:
similarly, `sequence != $3` not `sequence < $3` because we would still want to remove things if reverting to an earlier sequence

Reply:
(If I'm misremembering how rollbacks work and they create a new sequence # to promote to, this is irrelevant)